pg/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rubybench/ruby:0.5
+
+LABEL maintainer="https://github.com/bmarkons"
+
+ADD runner runner
+
+RUN chmod 755 runner
+RUN apt-get update && apt-get install -y \
+      rubygems \
```

Comment:
Hmm I thought rubygems is bundled together with ruby. So it is alittle strange that we have to install this dependency explicitly.